test/e2e/utils/TestControl.php

```diff
@@ -99,7 +111,15 @@ public function change_password($username, $password)
         return '';
     }
 
-    public function init_test_project($projectCode = null, $projectName = null, $ownerUsername = null)
+    public function reset_projects()
+    {
+        $db = MongoStore::connect(DATABASE);
+        $db->dropCollection('projects');
+        $db->createCollection('projects');
```

Comment:
AFAIK `$db->createCollection('projects');` is optional here.  The collection is created on first write.

Reply:
I think you're right. I'll remove it too.